### PR TITLE
LLT-5441 natlab preconditions

### PR DESCRIPTION
**Pull request by matislovas**
### Problem
Some nat-lab tests can be transfered as pre-conditions to running whole nat-lab test suite.

### Solution
Move `test_derp_client_message_forward` test to `pytest` pre-conditions.

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Review by Jauler — DISMISSED** [review]
+1.0

File: nat-lab/tests/conftest.py [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 from utils.vm import windows_vm_util, mac_vm_util
 
+DERP_SERVER_1_ADDR = "http://10.0.10.1:8765"
+DERP_SERVER_2_ADDR = "http://10.0.10.2:8765"
+DERP_SERVER_3_ADDR = "http://10.0.10.3:8765"
+DERP_SERVER_1_SECRET_KEY = "yBTYHj8yPlG9VtMYMwJSRHdzNdyAlVXGc6X2xJkjfHQ="
+DERP_SERVER_2_SECRET_KEY = "2NgALOCSKJcDxwr8MtA+6lYbf7b98KSdAROGoUwZ1V0="
+
+SETUP_CHECKS = [
+    ("setup_check_interderp", 10.0),
```
Comment (gytsto) [inline nat-lab/tests/conftest.py:20]: can't we just add function here, instead of name? wouldn't need to use `globals()` later on  :thinking:

Comment (matislovas) [reply nat-lab/tests/conftest.py:20]: I was having the impression, that it is not possible in python, but will try

Comment (matislovas) [reply nat-lab/tests/conftest.py]: Done

**Pull request merged**